### ISSUE TITLE
Blueprint collision error

### ISSUE THREAD

**Issue by ghost** [jeffknupp/sandman#33]
Output of `sandmanctl --all-columns-primary sqlite:////Users/jan/main.db`:

```
/Users/jan/.virtualenvs/foo/lib/python2.7/site-packages/sqlalchemy/ext/declarative/clsregistry.py:159: SAWarning: This declarative base already contains a class with the same class name and module name as flask_sqlalchemy.Chats, and will be replaced in the string-lookup table.
  existing.add_item(cls)
sandmanctl
Traceback (most recent call last):
  File "/Users/jan/.virtualenvs/foo/bin/sandmanctl", line 9, in <module>
    load_entry_point('sandman==0.7.0', 'console_scripts', 'sandmanctl')()
  File "/Users/jan/.virtualenvs/foo/lib/python2.7/site-packages/sandman/sandmanctl.py", line 58, in main
    activate(name='sandmanctl')
  File "/Users/jan/.virtualenvs/foo/lib/python2.7/site-packages/sandman/model/utils.py", line 155, in activate
    name)
  File "/Users/jan/.virtualenvs/foo/lib/python2.7/site-packages/sandman/model/utils.py", line 130, in register_classes_for_admin
    admin_view.add_view(admin_view_class(cls, db_session))
  File "/Users/jan/.virtualenvs/foo/lib/python2.7/site-packages/flask_admin/base.py", line 495, in add_view
    self.app.register_blueprint(view.create_blueprint(self))
  File "/Users/jan/.virtualenvs/foo/lib/python2.7/site-packages/flask/app.py", line 62, in wrapper_func
    return f(self, *args, **kwargs)
  File "/Users/jan/.virtualenvs/foo/lib/python2.7/site-packages/flask/app.py", line 885, in register_blueprint
    (blueprint, self.blueprints[blueprint.name], blueprint.name)
AssertionError: A blueprint's name collision occurred between <flask.blueprints.Blueprint object at 0x10af96610> and <flask.blueprints.Blueprint object at 0x10aef2390>.  Both share the same name "chatsview".  Blueprints that are created on the fly need unique names.
```

**Comment by jeffknupp** [comment]
This should be fixed in 0.7.1+ Could you retest?

**Comment by ghost** [comment]
Seems like it's fixed in 0.7.3, yes
